apps/tailwind-components/components/form/Field.vue

```diff
@@ -35,30 +35,54 @@ function validate(value: columnValue) {
 
   formFieldInput.value.validate(value);
 }
+
+const refData = ref();
+
+onMounted(async () => {
+  if (["ONTOLOGY", "ONTOLOGY_ARRAY"].includes(props.column.columnType)) {
```

Comment:
I don't want this here.

Reply:
meanwhile I don't mind this here temporarily if we have that mutual understanding.